SDK/AppCenter/Microsoft.AppCenter.UWP/Microsoft.AppCenter.UWP.csproj

```diff
@@ -11,11 +11,12 @@
     <AssemblyName>Microsoft.AppCenter</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.15063.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
```

Comment:
The main motivation for the work item was to work with a fresh installation of VS2019 with default SDK version (to avoid every contributor to download gigabytes of SDKs as separate steps).

The default SDK version that is currently installed (I checked with others here) is:

![Capture](https://user-images.githubusercontent.com/623672/58511309-60ec2900-814f-11e9-8e17-25544cb6aca4.PNG)

Please update the version.

Reply:
About upgrade v141 to v142, confirmed in only VS2019 scenario this is must have, otherwise MSBuild will throw an exception like below:
`error MSB8020: The build tools for Visual Studio 2017 (Platform Toolset = 'v141') cannot be found. To build using the v141 build tools, please install Visual Studio 2017 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution".`
However, if we do like this, then it should cannot be build on VS2017 only scenario. So, I guess that we'd better *keep v141, and install build tools for VS2017 on VS2019 box*.
@guperrot What you think about this way?

Reply:
Requesting our contributors to have 2 VS versions is defeating the purpose of this change. We already decided we were okay to migrate to VS 2019.